src/pages/settings/bank-accounts/edit/Edit.tsx

```diff
@@ -70,38 +74,38 @@ export function Edit() {
 
   const handleCancel = () => {
     if (!isFormBusy) {
-      navigate(route('/settings/bank_accounts'));
+      navigate('/settings/bank_accounts');
     }
   };
 
   const handleSave = async (event: FormEvent<HTMLFormElement>) => {
     if (!isFormBusy) {
-      event?.preventDefault();
+      event.preventDefault();
 
-      const toastId = toast.processing();
+      toast.processing();
       setErrors(undefined);
       setIsFormBusy(true);
 
-      try {
-        await request(
-          'PUT',
-          endpoint('/api/v1/bank_integrations/:id', { id }),
-          accountDetails
-        );
-        setIsFormBusy(false);
-        toast.success(t('updated_bank_account'), { id: toastId });
-        navigate(route('/settings/bank_accounts'));
-      } catch (cachedError) {
-        const error = cachedError as AxiosError;
-        console.error(error);
-        if (error?.response?.status === 422) {
-          setErrors(error?.response?.data?.errors);
-          toast.dismiss();
-        } else {
-          toast.error();
-        }
-        setIsFormBusy(false);
-      }
+      request(
+        'PUT',
+        endpoint('/api/v1/bank_integrations/:id', { id }),
+        accountDetails
+      )
+        .then(() => {
+          toast.success('updated_bank_account');
+          queryClient.invalidateQueries('/api/v1/bank_integrations');
+          navigate('/settings/bank_accounts');
+        })
+        .catch((error: AxiosError) => {
+          if (error?.response?.status === 422) {
```

Comment:
Please remove nullable reference to `error`.

Reply:
removed